api/v2beta1/helmrelease_types.go

```diff
@@ -691,6 +691,9 @@ type Test struct {
 	// actions in 'Install.IgnoreTestFailures' and 'Upgrade.IgnoreTestFailures'.
```

Comment:
This API will be deprecated and changes to `v2beta1` can be omitted.

Reply:
Done